README.md

```diff
@@ -36,6 +36,26 @@ We need to configure ```SocketIoModule``` module using the object ```config``` o
 
 Now we pass the configuration to the static method ```forRoot``` of ```SocketIoModule```
 
+### For Angular 6 Project
+
+ To be sure that you are using Angular 6, check your version:
+
+> ng -v
+
+If you are using Angular 6 you need change two thing
+
+1. Install rxjs-compat
+
+> npm install rxjs-compat
```

Comment:
@Mathias54
To my mind, it would be better if you add the `--save` flag for npm. Copy-paste will be a little bit faster 😃